rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala

```diff
@@ -101,6 +101,35 @@ class CollectPrinterSpec extends FlatSpec with Matchers {
     result shouldBe """{x0 : x1, 7 : "Seven"...free0}"""
   }
 
+  "Map" should "Print commas correctly" in {
```

Comment:
What does this have to do with commas? I thought it's about the element (key) order?

Reply:
True, but it had the effect that commas could be messed up and this is how we actually discovered the bug.